odoo-shippable/scripts/build-image.sh

```diff
@@ -423,7 +425,7 @@ for BASHRC in ${HOME}/.bashrc /home/odoo/.bashrc ${HOME}/.zshrc /home/odoo/.zshr
 do
     echo "Export the PYTHONPATH IN ${BASHRC}"
     cat >> $BASHRC << EOF
-if [ "x\${TRAVIS_PYTHON_VERSION}" == "x" ] ; then
+if [[ "x\${TRAVIS_PYTHON_VERSION}" == "x" ]] ; then
```

Comment:
@osvalr 
This change is necessary ? 
I made the test with this change and work well, is only to know if this change is required

Reply:
still pending for me to do an asciinema to showing you why this change.

Reply:
@JesusZapata in https://github.com/Vauxoo/docker-odoo-image/pull/287#issuecomment-346887763 we can see this behavior, zsh does not  take `if [ value1 == value2 ]; then` instead `if [[ value1 == value2 ]]; then` goes okay between zsh and bash